CONTRIBUTING.md

```diff
@@ -30,11 +30,13 @@ Our process for accepting changes operates by [Pull Request (PR)](https://help.g
 
         $ git checkout -b <branch-name>
 
-1.  **Create or modify the files** with your changes. If you want to show other people work that isn't ready to merge in, commit your changes then create a pull request (PR) with _WIP_ or _Work In Progress_ in the title.
+1.  **Create or modify the files** with your changes and commit them in Git. If you are fixing a known issue, include [“fixes #123” (where “123” is the issue number) in one of your commit messages](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords) to help automatically link everything together.
 
-        https://github.com/edgi-govdata-archiving/<repository-name>/pull/new/master
+1. Once your changes are ready for review, push your commits to GitHub and **[create a pull request (PR)](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork).** Assign as a reviewer or ping (using "`@<username>`") a Lieutenant (someone able to merge in PRs) active on the project (all Lieutenants can be pinged via `@edgi-govdata-archiving/lieutenants`).
```

Comment:
This was already in the README, but reading the sentence here, it's hard to understand with all the parenthetical's.  Suggest we update
```suggestion
1. Once your changes are ready for review, push your commits to GitHub and **[create a pull request (PR)](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork).** Assign as a reviewer or ping a Lieutenant active on the project. Lieutenants are able to merge in PRs. They can be pinged individually using "`@<username>`" or all Lieutenants can be pinged via `@edgi-govdata-archiving/lieutenants`.
```

Reply:
Yeah, I thought this was a bit awkward, too, but was trying to be relatively minimal.

What if we mostly just drop this part? I don’t think a non-member of a repo can assign reviewers, and I don’t think there’s much gained by having them try to figure out who to actually @-mention. People only do that approximately 1/100th of the time anyway, and I’m not sure it improves my workflow as the repo maintainer/person triaging issues & PRs. (Does having this in the initial PR post help you?)

> 1. Once your changes are ready for review, push your commits to GitHub and **[create a pull request (PR)](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork).**

Reply:
I also don’t think I’ve ever seen anyone @ the whole lieutenants group, and not sure it would be useful except as a last resort for a PR that’s been sitting with no feedback for weeks or something. (Most people in that group are not super active these days, and in general only one or two of the 20 of them would be a person relevant to any given repo.)

Reply:
Yea, I've never seen anyone use the Lieutenant thing either and few are active in that group. Let's axe it. 

The draft option info helps!

Reply:
DONE.